docs/application/dotnet/getting-started/runtime.md

```diff
@@ -0,0 +1,24 @@
+<a id="runtime"></a>
+# Managed Runtime Advantages
+
+A managed runtime offers the following advantages to your application development:
+
+- **Faster development**
+
+  Application development is accelerated since the managed runtime handles many functions that otherwise have to be coded into the application. The Tizen Native API and Tizen Web API already provide this benefit.
+
+- **Safer code**
+
+  Managed runtimes can handle, for example, bound checking, type safety, garbage collection, memory protection services, and objects being invoked directly. The Tizen Web API already provides some of these benefits, but not type safety, as JavaScript is not a strongly typed language.
+
+- **Lower deployment costs**
+
+  Component-based architecture makes it easier and faster to deploy applications in an enterprise environment characterized by multiple platforms, devices, and legacy systems.
+
+- **Better quality software**
+
+  Managed runtimes free you to focus on the business logic and code specific to the application, while reducing the number of coding errors.
+
+- **Cross-platform support**
+
+  The managed code is portable. Tizen .NET applications can have large portions of their logic applied to other systems supported by the .NET Core and Xamarin.Forms portions of the runtime.
```

Comment:
Rephrased:
The managed code is portable. Tizen .NET applications can have large portions of their logic applied to other systems supported by the .NET Core and Xamarin.Forms portions of the runtime.
to
The managed code is portable. Tizen .NET applications can have large portions of their logic applied to other systems supported by the .NET Core and Xamarin.Forms.

The last part is confusing in "supported by the .NET Core and Xamarin.Forms portions of the runtime", please verify.

Reply:
done. I remove it.